logtracing/span.go

```diff
@@ -9,12 +9,20 @@ import (
 	"time"
 )
 
+type SpanMeta struct {
```

Comment:
As for `func Meta()`, does this type need to be exported?

Reply:
Thanks, I'll update this.